packages/cxl-ui/package-lock.json

```diff
@@ -0,0 +1,4401 @@
+{
```

Comment:
Why is there a package-lock.json here suddenly?

Definitely use `yarn`, if you didn't, and send update with this file deleted.